plan.md

```diff
@@ -33,3 +33,21 @@ We should have ready supplies for 3-5 breakfasts, 2 lunches, 2 dinners. Ideally
 ### Snacks
 - 1 pack of cheese strings
 - 2 small bags of unsalted roasted nuts
+
+
+## Entertainment
+
+The entertainment will primarily be for eveining time as im sure there will be enough to do during the day. We will prepare some games, music and for some craic.
+
+### Games
+- Deck of playing cards
+- Scrabble
+- Connect 4
```

Comment:
I don't think that connect 4 is a good idea for a camping game. We'll lose some pieces for sure!